src/mds/MDSRank.cc

```diff
@@ -2914,6 +2914,8 @@ void MDSRankDispatcher::handle_asok_command(
   } else if (command == "cache status") {
     std::lock_guard l(mds_lock);
     mdcache->cache_status(f);
+  } else if (command == "quiesce subvolume") {
```

Comment:
and how do you unquiesce?

Reply:
Kill the op.

Reply:
There is a new "op kill" command as part of this series.

Reply:
I see it now with all commits selected 👍🏻